src/main/kotlin/no/nav/syfo/application/mq/MQSender.kt

```diff
@@ -19,14 +20,25 @@ interface MQSenderInterface {
 }
 
 class MQSender(private val env: Environment) : MQSenderInterface {
+
```

Comment:
Jeg har gjort noen endringer her slik at vi unngår å lage en ny connection til MQ for hver melding som skal sendes.